pkg/storage/database/database_driver.go

```diff
@@ -0,0 +1,23 @@
+package database
+
+type Driver interface {
+	// Returns the human-readable "name" of the driver.
+	Name() string
+
+	// Retrieves the blob matching provided digest.
+	GetBlob(digest string) (string, error)
+
+	// Uploads blob to database.
+	PutBlob(digest, path string) error
+
+	// Check if blob exists in database.
+	HasBlob(digest, path string) bool
+
+	// Delete a blob from the database.
+	DeleteBlob(digest, path string) error
+}
+
+type Blob struct {
+	Digest   string   `dynamodbav:"Digest,string"`
```

Comment:
Why use `dynamodbav` here? Isn't this generic?

Reply:
That is correct, I missed removing that.